### PR TITLE
add header and first section in the home page

### PR DESCRIPTION
**Pull request by rachaouldbabaali**
in this pull request I did the following changes :
- add the header to the navigation bar for the mobile version 
- set up linters and correcting errors
- add the headline secction

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+                      </a>
+                    </li>
+                    <li>
+                      <a href="#"
+                        ><img class="icon" src="images/finger.svg" alt="finger" />
+                      </a>
+                    </li>
+                  </ul>
+                </div>
+            </section>
```
Comment (Ibizugbe) [inline index.html:77]: Nice job on creating your headline section😉 but try to stick to the given design😉 as this is a requirement for this milestone😎 kindly see the image comparison below😉

| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|your headline|headline-mobile|

Kindly take note of the `font-properties`  and the `spacings` and the `background image` and make the necessary change😉

**Review by Ibizugbe — CHANGES_REQUESTED** [review]
Hi @rachaouldbabaali 👋👋, 

tryagain


Good job so far!👍
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights✅✅

- [x] Github flow practice has been used👍.
- [x] Linters are setup properly and they are passing👍.
- [x] The layouts for the content is done with flexbox👍.


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- Most of your commit messages look great but just a few of them are not capitalised😉 kindly ensure you capitalize your messages in future commit messages😉👍

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+                    d="M2.5 12a.5.5 0 0 1 .5-.5h10a.5.5 0 0 1 0 1H3a.5.5 0 0 1-.5-.5zm0-4a.5.5 0 0 1 .5-.5h10a.5.5 0 0 1 0 1H3a.5.5 0 0 1-.5-.5zm0-4a.5.5 0 0 1 .5-.5h10a.5.5 0 0 1 0 1H3a.5.5 0 0 1-.5-.5z"
+                  />
+                </svg>
+            </div>
+          </nav>
+        
+    </header>
+    <main>
+      <!--Headline section-->
+      <section class="headline-section">
```
Comment (Kamba56) [inline index.html:49]: - [ ] Great job adding fonts and font sizes to match the design. As requested by the previous reviewer, kindly fix all spacings too to match the design template. You are expected to stick to design as much as possible.
### Your design
image
### Figma template
image

File: index.html [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+            <ul>
+              <li>
+                <a class="logo" href="index.html">Racha</a>
+              </li>
+            </ul>
+            <ul class="links-holder">
+              <li class="link"><a href="index.html">Portoflio</a></li>
+              <li class="link"><a href="#">About</a></li>
+              <li class="link"><a href="#">Contact</a></li>
+            </ul>
```
Comment (Kamba56) [inline index.html:30]: - [Optional] Kindly consider using an indentation of two spaces in all your documents. This is a best practice and also a convention.

**Review by Kamba56 — CHANGES_REQUESTED** [review]
Hi @rachaouldbabaali, 

Great work on making the changes requested by a previous reviewer 👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- [x] Great use of Github workflow!👏👏👏
- [x] All linters passed!👏👏👏
- [ ] Great job using flexbox to layout contents

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Kamba56 — CHANGES_REQUESTED** [review]
Hi @rachaouldbabaali, 

Great work on making the changes requested by a previous reviewer 👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- [x] Great use of Github workflow!👏👏👏
- [x] All linters passed!👏👏👏
- [ ] Great job using flexbox to layout contents

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by divinecharlotte — APPROVED** [review]
# Hello @rachaouldbabaali       😃 🥳 🎉 🎈 

Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say other than... it's time to merge it 👍🏻 ✅ 
### To Highlight 💯

- No linters errors ✔️
- Descriptive PR ✔️
- Descriptive commit messages 👍 





## Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**